app.js

```diff
@@ -32,10 +30,10 @@ function initApp(options) {
     app.info = packageInfo;         // this app's package info
 
     // ensure some sane defaults
-    if (!app.conf.port) { app.conf.port = 8888; }
-    if (!app.conf.interface) { app.conf.interface = '0.0.0.0'; }
-    if (app.conf.compression_level === undefined) { app.conf.compression_level = 3; }
-    if (app.conf.cors === undefined) { app.conf.cors = '*'; }
+    if (!app.conf.port) { app.conf.port = 8888; } // eslint-disable-line max-statements-per-line
+    if (!app.conf.interface) { app.conf.interface = '0.0.0.0'; } // eslint-disable-line max-statements-per-line
+    if (app.conf.compression_level === undefined) { app.conf.compression_level = 3; }  // eslint-disable-line max-len,max-statements-per-line
+    if (app.conf.cors === undefined) { app.conf.cors = '*'; } // eslint-disable-line max-statements-per-line
```

Comment:
Or we could just change all of to to smth like `app.conf.port = app.conf.port || 8888`

Reply:
Wasn't sure whether I should touch the code. To me that is the better solution

Reply:
It is a super minor change in the code that will 100% not backfire, so I think it's ok

Reply:
Famous last words :) I'll make the change

Reply:
heh.. Actually, you're right. For `cors` undefined and false mean different things. My apologies.

Let's do the first 2 with `||` and second 2 just split the lines.